Tests/Unit/TokenProvider/ContentLocaleProviderTest.php

```diff
@@ -24,7 +25,8 @@ public function setUp()
         parent::setUp();
 
         $this->uriContext = $this->prophesize('Symfony\Cmf\Component\RoutingAuto\UriContext');
-        $this->provider = new ContentLocaleProvider();
+        $this->slugifier = $this->prophesize('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface');
```

Comment:
Absolutely no idea how this ever worked before.. slugifier was not set and we call `reveal()` on it below.